web/src/engine/websites/MangaLife.ts

```diff
@@ -0,0 +1,71 @@
+import { type Tag, Tags } from '../Tags';
+import icon from './MangaLife.webp';
+import { DecoratableMangaScraper, Manga, type MangaPlugin} from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON, FetchRequest, FetchWindowScript } from '../FetchProvider';
+
+export type APIManga = {
```

Comment:
There are many exports in this file, are these really used by other modules?

Reply:
Its just in case, but i can remove them.